src/setup/demo.md

```diff
@@ -0,0 +1,236 @@
+```
+
+In order to check that your local substra client can reach the demo server, you can use the following command:
+
+```sh
+# Quick test on the backend of the node-1
+curl substra-backend.node-1.com/readiness
+```
+
+You should receive an `"OK"` answer, if not please check again your `/etc/hosts` file. You can also use your browser to visit [substra-frontend.node-1.com](http:substra-frontend.node-1.com) & [substra-frontend.node-2.com](http:substra-frontend.node-2.com) (without https, because this is demo).
+
+### Login
+
+You can now configure your local Substra client, *with the credentials you received from us*, to connect to the `node-1` which hosts the organisation `org-1`:
+
+```sh
+# Configure the client, with a profile, a user, a password and an url
+substra config --profile <PROFILE> -u <USER> -p "<PASSWORD>" http://substra-backend.node-1.com
+
+# Login with the profile you created (located in your home folder: ~/.substra)
+substra login --profile <PROFILE>
+
+# You can then list the network nodes that are on the Substra network
+substra list node --profile <PROFILE>
+
+# And list, for example, the registered datasets in the node you are logged in
+substra list dataset --profile <PROFILE>
+```
+
+## 2. Follow Titanic example
+
+You will find everything you need to run a first end to end example with the tabular data of the Titanic: [repository](https://github.com/SubstraFoundation/substra/blob/master/examples/titanic/README.md).
+
+Please note that each resource is referenced by its hash in Substra, a unique identifier, so if you want to try out the ready to use Titanic example, you will need to edit, at least, the python files and the `description.md` for the assets (like adding a `print()` statement and editing the `description.md` files).
+
+## 3. Work with your own assets
+
+> **Note**: This is demo instance, please do not use any sensitive asset on this platform!
+
+The special [debugging section](https://github.com/SubstraFoundation/substra/tree/master/examples/debugging) can be off help preparing your asset, please have a look!
+
+### Data preparation
+
+The first step is partitioning your data, and create a test dataset and a train dataset. Following the Titanic example, you will first need to generate data samples (in `test_data_samples` and `train_data_samples` folders) by updating the `generate_data_samples` script with the path to you dataset and running it:
+
+```sh
+pip install -r scripts/requirements.txt
+python scripts/generate_data_samples.py
+```
+
+### Add dataset & objective
+
+You will now have to prepare the `opener.py` & `metrics.py` files, including their corresponding `description.md` and `Dockerfile` (where you have to add your depedencies). All assets are based on [substratools](https://github.com/SubstraFoundation/substra-tools/tree/master/substratools) classes to ease the process.
+
+The `add_dataset_objective.py` script is in charge of registering data to the `organisation`.
```

Comment:
You mean once you have finished preparing the opener and metrics files? Or independently and in parallel?

Reply:
It's in the order of the text: 
1. prepare the opener and metrics
2. edit the required elements in the script & run it

Reply:
OK, so for total clarity I would do:

```suggestion
The `add_dataset_objective.py` script is in charge of registering your dataset as an asset of the `organisation` you belong to. With your dataset will also be registered the `opener.py` & `metrics.py` files.
```